### PR TITLE
Add time window to vary delay between actions and fix helpers.get_following inconsistent ApiClient calls

### PR DESCRIPTION
**Pull request by tibotix**
Add `delay_tolerance_delta` attribute to set random time window when to perform the next action
Also added possibility to input user accounts manual to the bot by calling `from_user_list`

### CODE REVIEW

**Comment by stanvanrooy** [comment]
will check this tomorrow. Thanks for the contribution

**Comment by tibotix** [comment]
No problem. I have also changed one line in the ``helpers.get_following` function as there were two inconsistent calls to first `client.following_get` and second `client.followers_get` , which should be probably `client.following_get` too. 
Don't know if this was intended but feels a bit wrong.
Sry for my inexperience, i wanted to open another pull requests for this request but now its in this too...

File: instauto/bot/bot.py [hunk 1]
```diff
@@ -19,13 +19,14 @@ class Bot:
     _actions: List = []
     _post_cache: Dict[str, List[Dict]] = {}
 
-    def __init__(self, user_name: str, password: str, delay_between_action: float = 2.0) -> None:
+    def __init__(self, user_name: str, password: str, delay_between_action: float = 2.0, delay_tolerance_delta: float = 0.0) -> None:
```
Comment (stanvanrooy) [inline instauto/bot/bot.py:22]: could you rename to `delay_variance`? Seems a bit more clear.

Comment (stanvanrooy) [inline instauto/bot/bot.py:22]: could you rename to delay_variance? Seems a bit more clear.

File: instauto/bot/bot.py [hunk 2]
```diff
@@ -19,13 +19,14 @@ class Bot:
...
         """Initiate a new `Bot` instance.
 
         Args:
             user_name: the username of the account
             password: the password of the account
             delay_between_action: the amount of seconds to wait between actions (each like, follow, etc. is an action)
+            delay_tolerance_delta: the amount of seconds to tolerance variing of the delay_between_action (to bring more randomness)
```
Comment (stanvanrooy) [inline instauto/bot/bot.py:29]: "the amount of variance to add to the delay. Delay will be random number between (delay - variance) - (delay + variance).

File: instauto/helpers/friendships.py [hunk 3]
```diff
@@ -59,7 +59,7 @@ def get_following(client: ApiClient, user_id: str, limit: int) -> typing.List[di
             result.json()["users"]
         )
         logger.info("Retrieved {} of following, {} more to go.".format(len(following), limit - len(following)))
-        obj, result = client.followers_get(obj)
+        obj, result = client.following_get(obj)
```
Comment (stanvanrooy) [inline instauto/helpers/friendships.py:62]: Good catch!

**Review by stanvanrooy — CHANGES_REQUESTED** [review]

**Comment by tibotix** [comment]
Okay, all done. Sorry for my bad English..

**Comment by stanvanrooy** [comment]
@all-contributors please add @tibotix for code

**Pull request merged**